### ISSUE TITLE
Drag-n-drop not working

### ISSUE THREAD

**Issue by az-oolloow** [Gurenax/sfdx-lwc-fullcalendarjs#2]
First of all, many thanks for this amazing demo!

If one is looking simply to add full calendar, it's more than extensive.

unfortunately i have run into the issue where no matter what i tried, i couldn't get the `.draggable()` working with Full Calendar when hosted in LWC.

I can sort of get callbacks from html5 dragging but then i have to calculate start and stop myself, and the event hover is fixed to 2 hours always.

Have you ever tried this?

**Comment by Gurenax** [comment]
@az-oolloow 
I haven't played with the draggable functionality. It looks exciting though.

Is it this one?
https://fullcalendar.io/docs/v1/draggable

**Comment by az-oolloow** [comment]
@Gurenax Not really, i was talking about this particular sample: https://embed.plnkr.co/364sMw/

The only way i got it to work was using HTML5 drag n drop api which required me to bastardize a copy of fullcalendar, since it only supports jquery ui draggable out of the box.

**Comment by skammers** [comment]
Hey @az-oolloow, could you explain further this bastardize stuff? Ive been trying to get drag and drop to work for a while now. Im having the exact same problem as described in this post. The draggable element disappears whenever I start to drag. 

https://salesforce.stackexchange.com/questions/252027/jquery-ui-draggable
